lib/copy_peste/command/list_result.rb

```diff
@@ -0,0 +1,60 @@
+module CopyPeste
+  class Command
+    module ListResult
+
+      # Method used to display result list of module.
+      #
+      # @param result [Collection] collection data of result list
+      # @param name [String] Name of module, could be nil
+      def display_results(results, name)
+        results = (name && !name.empty?) ? results.where(:module_name => name.upcase) : results
+        unless results.empty?
+          @graph_com.display(10, "Module\t| Date\n------\t| ----")
+          results.each do |data|
+            @graph_com.display(10, "#{data.module_name}\t| #{data.created_at}")
+          end
+        else
+          @graph_com.display(12, "Option fail, name #{name} not found") 
```

Comment:
You should add the name of the option that failed like "Option [n] error:" because when we'll have more options it might be difficult to known which option failed.